### PR TITLE
Adding GSJUG as adoptor

### PR DESCRIPTION
**Pull request by mpredli01**
Good morning, everyone!

Let's try this again...

Mike.

### CODE REVIEW

File: data/spec_adopters.yml [hunk 1]
```diff
@@ -97,6 +97,14 @@ adopters:
       homepage_url: https://twitter.com/TurkeyJBUG/
       logo: turkeyjboss.png
       logo_white: turkeyjboss.png
+
+  - name: Jakarta NoSQL
+    directory: specifications/nosql/
+    adoptors:
```
Comment (ivargrimstad) [inline data/spec_adopters.yml:103]: There's a typo here. Should be adopters with an "e" not "o"...

**Review by ivargrimstad — CHANGES_REQUESTED** [review]
Just this little typo as far as I can see

**Comment by ivargrimstad** [comment]
And @mpredli01 Make sure you sign your commits. That's why the ECA-check fails

**Comment by autumnfound** [comment]
@mpredli01 @ivargrimstad closing this PR as there is another more up to date branch/PR

**Pull request closed without merging**